### DISCUSSION TITLE
Single Match Page Styling

### DISCUSSION THREAD

**Discussion by Zemill** [Heroes-Profile/heroesprofile#694] [Ideas]
For some reason, overall the page seems stretched left to right

http://127.0.0.1:8000/Match/Single/50721915

Need to add the header map image at the top.  I gave an example of that definition at the top of the page
https://github.com/Zemill/heroesprofile/blob/7712cc331a10af6280d6120f242d312d9292607c/resources/js/components/SingleMatch.vue#L13

I am not sure how to create a hover affect.   I attempted to replicate whats being done in RoundImage, but it didnt work.

The text at
https://github.com/Zemill/heroesprofile/blob/7712cc331a10af6280d6120f242d312d9292607c/resources/js/components/SingleMatch.vue#L144

needs to be put into a hover affect of the
https://github.com/Zemill/heroesprofile/blob/7712cc331a10af6280d6120f242d312d9292607c/resources/js/components/SingleMatch.vue#L147

And that font awesome icon needs styled and put in the correct location.  Like the production site

![image](https://github.com/Zemill/heroesprofile/assets/33404722/ae562fe0-a43f-4e8c-83dc-4e393e404f58)

**Comment by Zemill** [comment]
For reference, this is one page I am hoping to look almost exactly like the production site.

**Comment by lifeinabstract** [comment]
@Zemill  I think I fixed everything. Check and close this if it's good.

**Comment by Zemill** [comment]
@lifeinabstract 

Can you adjust the hover of the players.  Make the player more prominent

Production Site
![image](https://github.com/Zemill/heroesprofile/assets/33404722/5d8ff53d-f9e6-4e36-b609-530dbaa1ff0e)


Rewrite
![image](https://github.com/Zemill/heroesprofile/assets/33404722/00398961-d765-4901-a24e-72e689e16534)


For some reason I find the production site easier to read at a glance

**Comment by Zemill** [comment]
Also,
Can you move the party icons slightly up so they are partially outside the bubble?

**Comment by Zemill** [comment]
Getting duplication of background image I think

http://127.0.0.1:8000/Match/Single/50699534

![image](https://github.com/Zemill/heroesprofile/assets/33404722/1817b1c2-822a-4646-b654-6b4843f697ce)

**Comment by lifeinabstract** [comment]
updated again - let me know if i missed anything.

**Comment by Zemill** [comment]
We have

      <div class="bg-lighten">


For the different sections, but depending on whether it is esport, or if the data is available, sometimes we dont show entire sections.  So then it looks like two or more sections are merged.  Is there a way to make that class dynamically alternating regardless of whether we are showing a section or not?

For example
http://127.0.0.1:8000/Esports/NGS/Match/Single/17768

![image](https://github.com/Zemill/heroesprofile/assets/33404722/cd94b6da-725e-4338-9243-fd6720fc4bed)

**Comment by Zemill** [comment]
Also, can you move the patreon star up to be in the same line as the party icon?

http://127.0.0.1:8000/Match/Single/50847878

![image](https://github.com/Zemill/heroesprofile/assets/33404722/6c9446dc-15a4-4a17-8a66-5bafbe0c4377)

**Comment by lifeinabstract** [comment]
> We have
> 
> ```
>   <div class="bg-lighten">
> ```
> 
> For the different sections, but depending on whether it is esport, or if the data is available, sometimes we dont show entire sections. So then it looks like two or more sections are merged. Is there a way to make that class dynamically alternating regardless of whether we are showing a section or not?
> 
> For example http://127.0.0.1:8000/Esports/NGS/Match/Single/17768
> 
> ![image](https://private-user-images.githubusercontent.com/33404722/285905231-cd94b6da-725e-4338-9243-fd6720fc4bed.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDEzMDQ3MDQsIm5iZiI6MTcwMTMwNDQwNCwicGF0aCI6Ii8zMzQwNDcyMi8yODU5MDUyMzEtY2Q5NGI2ZGEtNzI1ZS00MzM4LTkyNDMtZmQ2NzIwZmM0YmVkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTMwVDAwMzMyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBlMjNiZDcxMWRiY2UxY2ZiZmU2ZWJkZWE2YWQ0ZGE4NDExMzExYTMzZTA4MWU4ODU3MmZjMGI4NmIwMTg3NTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.YyHG6TrNe54ZWMGk6mmFuxLM1Bl18fDX86_ksl80feg)

will look at this later - it's a bit involved to do.
